src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationevents/services/RegistrationEventsService.kt

```diff
@@ -26,19 +26,21 @@ class RegistrationEventsService(
   fun execute(hmppsDomainEvent: HmppsDomainEvent) {
     val registrationEventMessage: RegistrationAddedEventMessage = objectMapper.readValue(hmppsDomainEvent.message)
 
+    // TODO Autowire this in from application.yml
+    val baseUrl = "https://dev.integration-api.hmpps.service.justice.gov.uk"
```

Comment:
TODO?

Reply:
oops, missed that. Fixed in latest commit